tests/testthat/_snaps/io-csv-functions.md

```diff
@@ -1,4 +1,4 @@
-# arg raise_if_empty works
+# read/scan: arg raise_if_empty works
```

Comment:
I put the tests in io-csv-functions, where there are already the scan/read functionalities so I added this to separate the read/scan and the write tests.

On the other hand, tests for sink_csv are in test-lazyframe-frame.R. Maybe we could have "test-csv-read-scan.R" and test-csv-sink-write.R"?